spec/acceptance/trusted_external_data_spec.rb

```diff
@@ -54,7 +56,7 @@
 
     it "contains the node's CMDB record in the 'cmdb_ci' table obtained by querying the 'fqdn' field" do
       result = trigger_puppet_run(master)
-      trusted_json = parse_json(result.stdout, 'trusted_json')
+      trusted_json = parse_trusted_json(result.stdout)
```

Comment:
I don't see `parse_trusted_json` being defined anywhere.

Reply:
Yeah it looks like you've changed this function name to just `parse_json` and this invocation didn't get changed to `parse_json(result.stdout, 'trusted_json')` during the rebase like the others did.

Reply:
Yeah sorry, that was before your rebase.